static/js/tools/map/app.tsx

```diff
@@ -79,10 +85,16 @@ function App(): ReactElement {
           <Row>
             <PlaceOptions toggleSvHierarchyModal={toggleSvModalCallback} />
           </Row>
-          <Row>{useStandardizedUi ? <VisToolInstructionsBox /> : <Info />}</Row>
-          <Row id="chart-row">
-            <ChartLoader />
-          </Row>
+          {showInstructions && (
+            <Row>
+              {useStandardizedUi ? <VisToolInstructionsBox /> : <Info />}
+            </Row>
+          )}
```

Comment:
You'll probably get merge conflicts with these lines I just submitted: https://github.com/datacommonsorg/website/blob/dc528274a51601af92e68f8353b0f71c12ddb14d/static/js/tools/map/app.tsx#L85-L91

Between the two, I prefer yours both for readability and so the `showInstructions` logic also gates the prod facing `<Info>` component.